src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/blocks/media/fixtures.js

```diff
@@ -34,7 +34,7 @@ export const CPSMediaBlock = {
 };
 
 export const optimoVideoBlock = {
-  type: 'video',
+  type: 'audio_video',
```

Comment:
:O This is in the wrong place, its `format: 'video',` in `model` that needs changing

Reply:
damn - this is what I get for making commits in meetings - thanks for noticing - updated!